Frontend/src/app/Models/batch.model.ts

```diff
@@ -0,0 +1,22 @@
+export class Batch {
+
+    // BatchId: string;
```

Comment:
As a good practice we do not add commented codes but it depends.

Reply:
Oh,my mistake. I forgot to commit the updated batch model. In there I deleted this.